temporal-testing/src/main/java/io/temporal/internal/testservice/TestWorkflowMutableStateImpl.java

```diff
@@ -2115,6 +2126,209 @@ public void completeQuery(QueryId queryId, RespondQueryTaskCompletedRequest comp
     }
   }
 
+  @Override
+  public DescribeWorkflowExecutionResponse describeWorkflowExecution() {
+    WorkflowExecutionConfig.Builder executionConfig =
+        WorkflowExecutionConfig.newBuilder()
+            .setTaskQueue(startRequest.getTaskQueue())
+            .setWorkflowExecutionTimeout(startRequest.getWorkflowExecutionTimeout())
+            .setWorkflowRunTimeout(startRequest.getWorkflowRunTimeout())
+            .setDefaultWorkflowTaskTimeout(startRequest.getWorkflowTaskTimeout());
+
+    GetWorkflowExecutionHistoryRequest getRequest =
+        GetWorkflowExecutionHistoryRequest.newBuilder()
+            .setNamespace(startRequest.getNamespace())
+            .setExecution(executionId.getExecution())
+            .build();
+    List<HistoryEvent> fullHistory =
+        store
+            .getWorkflowExecutionHistory(executionId, getRequest, null)
+            .getHistory()
+            .getEventsList();
+
+    WorkflowExecutionInfo.Builder executionInfo = WorkflowExecutionInfo.newBuilder();
+    executionInfo
+        .setExecution(this.executionId.getExecution())
+        .setType(this.getStartRequest().getWorkflowType())
```

Comment:
There is some mix of styles here. Accessing private fields through this, without this, using getters. It looks like this class is written mostly using the second style, so let's normalize code style in these new methods.